src/views/Home.jsx

```diff
@@ -1,11 +1,12 @@
 import './Home.css';
 
-export function Home() {
+export function Home({ handleClick }) {
 	return (
 		<div className="Home">
 			<p>
 				Hello from the home (<code>/</code>) page!
 			</p>
+			<button onClick={handleClick}>Create list</button>
```

Comment:
This button should be hidden if a `listToken` already exists :slightly_smiling_face:

Reply:
if the user has a token, gets redirected to the List view, so the user won't see the button. Isn't it correct that way?

Reply:
Just fixed this!

Quick question! Do users need to be redirected to the List view from wherever they are in the app or does this behavior only pertains to the Home view?

Reply:
@thatgirldorian has already fixed it :)